pkg/delete/internal/segment/delete.go

```diff
@@ -79,7 +79,7 @@ func deleteSingle(ctx context.Context, c client, dp pointer.DeletePointer) error
 }
 
 func findEntryWithExternalID(ctx context.Context, c client, dp pointer.DeletePointer) (string, error) {
-	items, err := list(c, ctx)
+	items, err := list(ctx, c)
```

Comment:
Same here, please refactoring into separate tickets

Reply:
Moved to https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1723